content/blog/2022-03-16-march-22-heartbeat.md

```diff
@@ -0,0 +1,300 @@
+---
+title: March '22 Heartbeat
+date: 2022-03-16
+description: >
+  Monthly updates are here! You will find great tutorials and workflows from  
+  the Community, Online course updates, MLOps maturity models, new docs and
+  more! Happy Spring!
+
+descriptionLong: |
+  This month you will find:
+
+    🧘🏻‍♀️ MLOps is a mess, but that's ok,
+    
+    🥰 Tutorials and workflows from the Community,
+
+    🗣 Upcoming Events,
+
+    🔺 MLOps Maturity Models
```

Comment:
Should a comma be here like with the other topics?

Reply:
Yep, thanks!